op-challenger/game/scheduler/coordinator.go

```diff
@@ -133,7 +135,10 @@ func (c *coordinator) createJob(ctx context.Context, game types.GameMetadata, bl
 			return nil, fmt.Errorf("failed to create game player: %w", err)
 		}
 		if err := player.ValidatePrestate(ctx); err != nil {
-			return nil, fmt.Errorf("failed to validate prestate: %w", err)
+			if !c.allowInvalidPrestate {
+				return nil, fmt.Errorf("failed to validate prestate: %w", err)
+			}
+			c.logger.Error("Invalid prestate", "game", game.Proxy, "err", err)
```

Comment:
nit: This skips over all validation errors, including RPC-related errors. Probably better to only ignore specifically prestate mismatches.

Reply:
The RPC calls are only done to check the prestate which we'd ignore but yeah its probably worth being picky so we test as much of the code path as we can in CI.